html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Welcome to my Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="logo-container">
+      <a class="text-logo" href="#">Yassine</a>
+    </div>
+    <nav class="nav-menu">
+      <img src="/images/Union.png" alt="menu icon">
+    </nav>
+  </header>
+  <main class="headline">
+      <div class="titel-container">
+        <h2 class="main-titel">Hey there. I’m Yassine </h2>
+        <h2 class="secondary-titel">
+          I’m a software developer
+        </h2>
+      </div>
+      <div class="des-text-container">
+      <p class="description-text">
+         I believe that i can use my skills to help you create cool products and features , you take a look at my projects if you like what you see
+         have a project you need coded,
+         contact me to talk about it
+      </p>
+      </div>
+      <div class="social-media-icons">
+        <a href="https://www.twitter.com/yf_omari" class="twitter-link"><img src="/images/twitter.svg" alt="twitter icon"></a>

```

Comment:
- [ ] I think it's a good idea to use the `<ul>` tag (unordered list) in this situation when listing items/links. [Here you can find something about "ul" tags](https://www.w3schools.com/tags/tag_ul.asp) with examples. Please use `<ul>` tag to wrap the links and don't forget to use the `<li>` tag (list item) for each item in your list :memo: